src/index.js

```diff
@@ -0,0 +1,185 @@
+import './scss/style.scss'
+import { async } from 'q'
+
+const API_KEY = '3df9465bacaddb460ad6f26f2a256db2'
+const API_URL = 'https://api.openweathermap.org/data/2.5/weather?units=metric'
+const API_IMG_URL = 'http://openweathermap.org/img/wn/'
+const API_NEXTWEEK_URL = 'https://api.openweathermap.org/data/2.5/forecast?units=metric'
+let API_COUNTRY_CODE = 'pl'
+
+const UNITS = {
+  pressure: 'hPa',
+  humidity: '%',
+  wind: 'km/h',
+  temperature: '°C'
+}
+
+const DOW = {
```

Comment:
`DOW` ? Zgaduje że chodziło o `Day of week` ? 😃 Jeśli tak to spokojnie zamiast skrótu możemy użyć pełnej nazwy :)